### ISSUE TITLE
[RFE] Add capabilities to mask IP and other data throughout a series of SOSReports

### ISSUE THREAD

**Issue by testingillusions** [soscleaner/soscleaner#163]
**Is your feature request related to a problem? Please describe.**
When looking to send logs from multiple systems, it would be nice to have the SOS Cleaner mask all the IP addresses the same across the set to allow for easier tracing between systems. 

**Describe the solution you'd like**
Allow SOS Cleaner to take in a directory of SOSReports to cleanse them all at the same time with the same values.

For instance, there are 2 SOS Reports with access_log; log1 and log2

Original data:
Log1: 10.185.248.71 - - [09/Jan/2015:19:12:06 +0000] 808840 "GET  /inventory" 500 17 "-" 
Log2: 10.185.248.71 - - [09/Jan/2015:19:12:06 +0000] 808840 "GET  /inventory" 500 17 "-" 

Cleansed Data
Log1: 99.123.456.20 - - [09/Jan/2015:19:12:06 +0000] 808840 "GET  /inventory" 500 17 "-" 
Log2: 99.123.456.20 - - [09/Jan/2015:19:12:06 +0000] 808840 "GET  /inventory" 500 17 "-" 

**Describe alternatives you've considered**
None that we can think of at this time.

**Additional context**
None

**Comment by slashdd** [comment]
It's seems to be in the same vein as issue #150

**Comment by testingillusions** [comment]
@slashdd , it would be very similar. This addresses doing a batch of SOS Reports, however, #150 would likely leverage the same modification.

My thought here would be to create a file that holds the mapping. Only concern would be how to clean up the this file as it may be a security concerns.

**Comment by slashdd** [comment]
Right, we are currently working at porting soscleaner to py3. We could then try to look at it once the py3 is ready and found in master branch.

**Comment by jduncan-rva** [comment]
This wouldn't be hard to do in the short term by ingesting and then adding to the same generated reports. 

However I'd like to leverage this in such a way to make it friendlier to expose soscleaner down the road through an API.
